src/proposal.mligo

```diff
@@ -285,17 +285,6 @@ let set_voting_period(new_period, config, store : voting_period * config * stora
   let store = { store with voting_period = new_period; last_period_change = vp_log } in
   (([] : operation list), store)
 
-// Update quorum_threshold. The new quorum_threshold affects
-// all ongoing and new proposals.
-[@inline]
-let set_quorum_threshold(new_threshold, config, store : quorum_threshold * config * storage): return =
-  let store = authorize_admin store in
-  if   is_le_qt (new_threshold, config.max_quorum_threshold)
```

Comment:
The definition of `is_le_qt` can now also be removed, as this was its only use.

Reply:
Updated